tests/test_wave_02.py

```diff
@@ -1,7 +1,7 @@
 import pytest
 from viewing_party.main import *
 
-
+"""
```

Comment:
I'm curious why you have these tests commented out. They seem to pass when I uncomment them